src/app/backend/auth/jwe/keyholder.go

```diff
@@ -136,7 +138,7 @@ func (self *rsaKeyHolder) getEncryptionKeyHolder() runtime.Object {
 	priv, pub := ExportRSAKeyOrDie(self.Key())
 	return &v1.Secret{
 		ObjectMeta: metaV1.ObjectMeta{
-			Namespace: authApi.EncryptionKeyHolderNamespace,
```

Comment:
AFAICT EncryptionKeyHolderNamespace is used in a few other places. Like auth/api/common.go looks like it is preventing dashboard from overwriting it's own config. Have those been fully addressed?

Reply:
@FrenchBen This looks to be the last dangling thing before we can lgtm this. 

Several files under auth still reference EncryptionKeyHolderNamespace that should become dynamic. Happy to work with you to move this along if you need any help. 👍 Thanks!